Dockerfile

```diff
@@ -74,6 +75,18 @@ RUN mkdir -p ${APP_HOME}
 WORKDIR ${APP_HOME}
 # End
 
+# Download and install filebeat for sending logs to logstash
+ENV FILEBEAT_VERSION=7.6.2
+RUN apk add --update-cache curl && \
+    curl https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-${FILEBEAT_VERSION}-linux-x86_64.tar.gz -o filebeat.tar.gz && \
```

Comment:
Would be good to compare a checksum if possible. See what we did for the Verify image.

Reply:
Yea that's a good point, I've added in the checksum. Thanks F